crates/simulacrum/src/lib.rs

```diff
@@ -468,7 +468,7 @@ impl<T, V: store::SimulatorStore> ReadStore for Simulacrum<T, V> {
         todo!()
     }
 
-    fn get_lowest_available_checkpoint(
+    fn get_lowest_available_checkpoint_for_state_sync(
```

Comment:
is get_lowest_available_checkpoint_for_state_sync now any different from get_lowest_available_checkpoint?
do we need to keep both methods?

Reply:
ah, thank you for catching this. I'll revert that change